pkg/api/server/v1alpha2/records.go

```diff
@@ -63,6 +62,7 @@ func (s *Server) CreateRecord(ctx context.Context, req *pb.CreateRecordRequest)
 	// Populate Result with server provided fields.
 	protoutil.ClearOutputOnly(r)
 	r.Id = uid()
+	r.Uid = r.Id
```

Comment:
good catch here. Looks like this was a lingering bug because the `.Id` field is marked "deprecated"